com.adrienplazas.Metronome.json

```
@@ -0,0 +1,98 @@
+{
+    "app-id": "com.adrienplazas.Metronome",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "metronome",
+    "finish-args" : [
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/metronome/cargo",
+            "RUSTFLAGS" : "-L=/app/lib"
```

```suggestion
            "CARGO_HOME" : "/run/build/metronome/cargo"
```
this was only needed at times where you needed to build gtk4 yourself and was used to fix a pango linking issue. Can be safely dropped

---

I assume Solanum should be updated then, this is where I copied it from: https://github.com/flathub/org.gnome.Solanum/blob/master/org.gnome.Solanum.json#L21